omnibus/files/private-chef-cookbooks/private-chef/libraries/preflight_postgres_validator.rb

```diff
@@ -186,7 +186,7 @@ def backend_verify_database_access(connection)
   def backend_verify_postgres_version(connection)
     # Make sure the server is a supported version.
     r = connection.exec('SHOW server_version;')
-    v = Gem::Version.new(r[0]['server_version'])
+    v = Gem::Version.new(r[0]['server_version'].to_f)
```

Comment:
this works as long as postgres does x.y versions but will fail if they move to x.y.z

you might want to regexp match on `/^([0-9\.]+)/` instead to future proof it a bit?

Reply:
Our requirements hopefully don't hit the patch version so I think we should be good there, but it's worth putting a comment in to note the limitation

Reply:
Good suggestion.  I'll try it.